.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+
+updates:
+
+  # Keep dependencies for GitHub Actions up-to-date
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
Maybe `daily` is too aggressive as a default for every track repo?

For the test runner repos, it can be important to have fast security fixes. But perhaps it's less important for the track CI.

That said, track repos might generally use `foo/bar@v1` or `foo/bar@master` for the action versions anyway, rather than pinning to some hash (which is generally more important for the repos that end up exposing an attack surface). And so maybe then the ones that **are** pinned to a hash should be updated quickly.

Reply:
🤷 My reasoning for the daily schedule was that we would have access to security fixes soon. It is not likely that the actions that are used change very frequently, so I _think_ this might not actually be a big issue. But I'll ask in v3.